### ISSUE TITLE
Slice/Elevate error w/ issue parallel engine

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1536]
cq-id: VisIt00009021
cq-submitter: Cyrus Harrison
cq-submit-date: 02/11/10

Gary Kerbel called and reported a problem using slice + elevate on alastor in parallel.

Data @ /g/g24/cyrush/visit/ticket009021 on OCF.

Add PC Plot of Jz
Slice on Z Axis
Elevate rel to xy limits. 

When using 16 processors on alastor, the correct plot is produced.
When using 32 a large hole appears in the plot, it looks like it is an extents used to elevate are incorrect for a subset of the problem.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 06/21/2010 07:14 pm
Original update: 01/05/2011 02:47 pm
Ticket number: 26
